__tests__/user/user.mutation.test.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-undef */
+require('dotenv').config();
```

Comment:
Please, remove this line, it is going to use env config for development. Why have you added this line?

Reply:
Fixed